src/Stripe.net/Services/Account/StripeAccountCreateOptions.cs

```diff
@@ -7,7 +7,7 @@ public class StripeAccountCreateOptions : StripeAccountSharedOptions
         [JsonProperty("country")]
         public string Country { get; set; }
 
-        [JsonProperty("managed")]
-        public bool? Managed { get; set; }
+        [JsonProperty("type")]
+        public string Type { get; set; }
```

Comment:
I'm a bit hesitant about the use of the attribute `Type`.  To avoid future namespace issues, it might make sense to change this to something like `AccountType`.  I'm pretty soft on this feedback though, as I don't know if it could have any weird namespace conflicts.